lib/csv.rb

```diff
@@ -1741,20 +1765,28 @@ def headers
     end
   end
 
+  # :call-seq:
+  #   csv.return_headers?
   # Returns the value that determines whether headers are to be returned; used for parsing;
   # see {Option +return_headers+}[#class-CSV-label-Option+return_headers]:
   #   CSV.new('').return_headers? # => false
   def return_headers?
     parser.return_headers?
   end
 
+  # :call-seq:
+  #   csv.write_headers? -> object
```

Comment:
```suggestion
  #   csv.write_headers? -> true or false
```

We should change the default value of `write_headers` to `false` from `nil`.